libs/postgresql/Makefile

```diff
@@ -168,13 +167,11 @@ define Host/Install
 	$(INSTALL_BIN) $(HOST_BUILD_DIR)/src/port/libpgport.a $(STAGING_DIR_HOSTPKG)/lib/
 	$(INSTALL_BIN) $(HOST_BUILD_DIR)/src/bin/pg_config/pg_config $(STAGING_DIR_HOSTPKG)/lib/
 	$(INSTALL_DIR) $(STAGING_DIR_HOSTPKG)/bin/
-	$(INSTALL_BIN) $(HOST_BUILD_DIR)/src/interfaces/ecpg/preproc/ecpg $(STAGING_DIR_HOSTPKG)/bin/
```

Comment:
Maybe we could keep ecpg installed to STAGING_DIR_HOSTPKG as it may be used and expected to be available by projects making use of PostgreSQL. However, I'm not aware of any...